pkgs/by-name/ho/horcrux/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

I ran this new package through `nix run github:NixOS/nixfmt` and the following changes below were made. Might be worthwhile to use what will likely be the standard format. What do you think, @markus-heinrich?
```suggestion
{
  lib,
  buildGoModule,
  fetchFromGitHub,
}:
```

---

Sure, sounds good! I wasn't aware of `nixfmt`, thanks for the hint 🙂